### DISCUSSION TITLE
statdata 데이터 팩키지 마무리

### DISCUSSION THREAD

**Discussion by statkclee** [tidyverse-korea/statdata#27] [Ideas]
`statdata` 팩키지가 이런 저런 사정으로 현재 답보상태라 이를 타계하기 위해서 의견을 모아 마무리를 해야 될 듯 싶습니다.
데이터에 대해서는 멘토와 멘티가 각자 데이터를 모아 주셔서 어느 정도 마무리가 되었지만... 나머지 패키지를 마무리하는데 필요한 로고, 웹사이트, 소품(Vignette) 등에서는 마지막 마무리 작업이 되어 version 1.0을 찍고 각자 추억으로 남기고 다음을 기약해야 될 듯 싶습니다. 의견 계시면 부탁드립니다.

**Comment by statkclee** [comment]
추석 기준을 데이터셋에 대한 baseline 을 긋고, `statdata` 팩키지 로고 제작하고 `pkgdown`으로 웹사이트 간단히 제작하고 사용법에 대한 tutorial Rmd 파일 3개 제작하고 README 파일 보강하였습니다. 
이 상태에서 밝혀진 문제점을 보완하는 것으로 `statdata` 팩키지에 대해 기본 작업을 마무리하고자 합니다.
특히, 윈도우와 맥에서 데이터 팩키지 제작에 한글 깨짐 문제 뿐만 아니라 윈도우와 맥에서 `statdata` 팩키지 사용할 때 한글 깨짐에 대해서도 정리를 해둘 필요가 있겠습니다.

@HwayoungYoon 한국 R 컨퍼런스 발표 때 좋은 떡밥이 되지 않을까 싶습니다. 😄

**Comment by choonghyunryu** [comment]
@statkclee 
한글 깨짐에 대해서는 정리해 둔 것이 있습니다. 마무리가 덜 되었지만, 금주 중으로 마무리 해 보겠습니다.

https://choonghyunryu.github.io/develop_Korean_package/
